manifests/charts/base/templates/validatingwebhookconfiguration.yaml

```diff
@@ -15,7 +15,9 @@ webhooks:
       {{- else }}
       service:
         name: istiod
+        {{- if .Values.global.istioNamespace }}
```

Comment:
this field is required.

Reply:
Not sure If I updated it or it gets updated during `make gen`. But I will verify.